tests/utils/test_credentials.py

```diff
@@ -66,8 +66,15 @@ def test_read_mlflow_creds_file(tmp_path, monkeypatch):
         ("username", None),
         (None, "password"),
         (None, None),
-    ],
-)
+    ])
+@pytest.mark.parametrize(
+    ("user", "7<{=.=@&{8[.0_+$|`}16\%2"),
+    [
+        ("user", "7<{=.=@&{8[.0_+$|`}16\%2"),
+        ("user", None),
+        (None, "7<{=.=@&{8[.0_+$|`}16\%2"),
+        (None, None),
+    ])
```

Comment:
```suggestion
        ("username", "7<{=.=@&{8[.0_+$|`}16\%2"),
        (None, "7<{=.=@&{8[.0_+$|`}16\%2"),
    ],
)
```

We can just add to the existing parametrized collection.